manifests/client.pp

```diff
@@ -49,13 +49,11 @@
     $file_ensure = 'absent'
   }
 
-  if $facts['os']['family'] != 'Suse' {
-    if $package_name != undef {
-      package { 'snmp-client':
-        ensure => $package_ensure,
-        name   => $package_name,
-        before => File['snmp.conf'],
-      }
+  if ($package_name != undef) and ($package_name != $snmp::package_name) {
```

Comment:
Can move this in a dedicated PR with a clear title ?

Reply:
No; I believe the title and explanation are clear.